website/config/environment.js

```diff
@@ -30,7 +30,10 @@ module.exports = function (environment) {
         'Helios Design System provides the building blocks to design and implement consistent, accessible, and delightful product experiences across HashiCorp.',
       siteName: 'Helios Design System',
       title: 'Helios Design System',
-      imgSrc: '/assets/logos/share-card.jpg',
+      // TESTING below!!! was: '/assets/logos/share-card.jpg'
+      imgSrc:
+        'https://assets.codepen.io/133490/mike.jpg',
```

Comment:
How should I actually do the real share-card image full url?

Reply:
it has to be the full URL because it is not relative to the social media site, if that makes sense.

Reply:
@KristinLBradley I have an idea on how we can solve this. Let's pair for a few minutes when you have time.

Reply:
but basically my thought is we can concatenate `imgSrc` and `url` where necessary to give the proper path to the image.

Reply:
I've been googling around and trying to figure out how to grab the site url but still not sure how to go about it.

Reply:
let's pair on it tomorrow